scripts/asm-installer/install_asm

```diff
@@ -718,6 +733,17 @@ EOF
     info "Downloading ASM kpt package..."
     retry 3 run kpt pkg get "${KPT_URL}" asm
   fi
+
+  while read -d ',' -r yaml_file; do
+    yaml_file="${OPTIONS_DIRECTORY}/${yaml_file}.yaml"
+    if [[ ! -f "${yaml_file}" ]]; then
+      fatal "Couldn't find remote yaml file ${yaml_file}."
```

Comment:
Maybe list the options users can choose here?

Reply:
Good idea.